data/net/minecraft/realms/RealmsScreen.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/realms/RealmsScreen
+	METHOD addLabel (Lnet/minecraft/realms/RealmsLabel;)Lnet/minecraft/realms/RealmsLabel;
+		ARG 1 label
+	METHOD row (I)I
+		ARG 0 offset
```

Comment:
I'm not sure if `offset` is the best param name here. While yes, it offsets from that particular hardcoded position,  it's used more like a position. Offsets are usually more indicative of loops to which this method never seems to be used and is also supplied negative values at times.